SIS_import/sis_upload.rb

```diff
@@ -321,6 +326,11 @@ def prior_upload_error
 		end
 
 		process_result = Canvas_API_GET("#{$server_api_url}accounts/#{ACCOUNT_NUMBER}/sis_imports/#{process_id}")
+		if (process_result.nil?)
+			## cannot find status info for prior upload job
```

Comment:
All Canvas call modules (Canvas_API_GET, Canvas_API_POST, Canvas_API_PUT) use the json_parse_safe() in utils.rb file. Proper loggings at the error level are done in that file when null JSON value is encountered.